src/PerspectivePanel.test.tsx

```diff
@@ -100,7 +100,7 @@ describe('PerspectivePanel', () => {
     expect(load).toHaveBeenCalledTimes(2);
 
     panel.setProps({
-      data: { series: [] } as PanelData,
+      data: ({ series: [] } as unknown) as PanelData,
```

Comment:
Why two `as`?

Reply:
compile error without it -- says that `never[]` array can not be made into PanelData.sereis